.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
```

Comment:
this does not match with the rules set in `prettierrc.json`

Reply:
fixed

related commit - https://github.com/the-ai-team/razor/commit/1c04b19bd37ac93a17e362f3e71f53c4bf7c2bf3